prairielib/lib/util.js

```diff
@@ -22,3 +22,13 @@ module.exports.sanitizeObject = function sanitizeObject(value) {
         return value;
     }
 };
+
```

Comment:
@mwest1066 

It seems like this file is the common area for these sort of helper functions. However, either this is duplicate code, or you are moving away from using the Prairie Learn node module. 

The `require('@prairielearn/prairielib').util` contains the same `sanitizeObject` function seen in the `prairielearn/prairielib/util.js` file. 

Please let me know where you want this code.

Reply:
![image](https://user-images.githubusercontent.com/7872295/102383676-aca83280-3f80-11eb-863e-d49d8f7dc069.png)

I'm seeing a lot of use of the node module.

Reply:
*Asked in Slack. Library approach is deprecated.